src/hugo/content/en-us/implementations/aws/deviceOnboarding/Provisioning-JITP/_index.md

```diff
@@ -0,0 +1,143 @@
+
+ACM Private CA operations can be accessed from the AWS Management Console, using the ACM Private CA API, or using the AWS CLI.
+
+#### Register Certificate Authority with AWS IoT Core
+
+To set up a JITP environment with AWS IoT Core, first register your CA (Certificate Authority) with AWS IoT Core.
+You can either [register a self-managed CA](https://docs.aws.amazon.com/iot/latest/developerguide/register-CA-cert.html) or [register ACM managed private CA](https://iot-device-management.workshop.aws/en/provisioning-options/bring-your-own-ca.html) with AWS IoT.
+
+---
+#### Create and register a provisioning template
+---
+A [provisioning template](https://docs.aws.amazon.com/iot/latest/developerguide/provision-template.html) is a JSON document that uses parameters to describe the resources your device must use to interact with AWS IoT. IoT Core will use this template to register a new device with certificate signed by a registered CA.
+
+1. Create an IAM role for your AWS IoT Core service and name it JITP_demo. Attach Policy AWSIoTThingsRegistration to this role.
+![iam](iam.png)
+2. Create a JITP template JSON file by running following commands
+```json
+sudo nano jitp_template.json
+```
+3. Paste the following serialized JSON and save the file using control + X and type Y
+
+`Replace <REGION>, <Account_ID> and <ROLE_ARN> with your AWS Region, Account_ID and Role ARN for JITPRole respectively`
+```json
+{
+ "templateBody":"{ \"Parameters\" : { \"AWS::IoT::Certificate::CommonName\" : { \"Type\" : \"String\" },\"AWS::IoT::Certificate::Country\" : { \"Type\" : \"String\" }, \"AWS::IoT::Certificate::Id\" : { \"Type\" : \"String\" }}, \"Resources\" : { \"thing\" : { \"Type\" : \"AWS::IoT::Thing\", \"Properties\" : { \"ThingName\" : {\"Ref\" : \"AWS::IoT::Certificate::CommonName\"}, \"AttributePayload\" : { \"version\" : \"v1\", \"country\" : {\"Ref\" : \"AWS::IoT::Certificate::Country\"}} } }, \"certificate\" : { \"Type\" : \"AWS::IoT::Certificate\", \"Properties\" : { \"CertificateId\": {\"Ref\" : \"AWS::IoT::Certificate::Id\"}, \"Status\" : \"ACTIVE\" } }, \"policy\" : {\"Type\" : \"AWS::IoT::Policy\", \"Properties\" : { \"PolicyDocument\" : \"{ \\\"Version\\\": \\\"2012-10-17\\\", \\\"Statement\\\": [ { \\\"Effect\\\": \\\"Allow\\\", \\\"Action\\\": [ \\\"iot:Connect\\\" ], \\\"Resource\\\": [ \\\"arn:aws:iot:<REGION>:<ACCOUNT_ID>:client\\\/*\\\" ] }, { \\\"Effect\\\": \\\"Allow\\\", \\\"Action\\\": [ \\\"iot:Publish\\\", \\\"iot:Receive\\\" ], \\\"Resource\\\": [ \\\"arn:aws:iot:<REGION>:<ACCOUNT_ID>:topic\\\/*\\\" ] }, { \\\"Effect\\\": \\\"Allow\\\", \\\"Action\\\": [ \\\"iot:Subscribe\\\" ], \\\"Resource\\\": [ \\\"arn:aws:iot:<REGION>:<ACCOUNT_ID>:topicfilter\\\/*\\\" ] } ] }\" } } } }",
+ "roleArn":"<ROLE_ARN>"
+}
+```
+4. Run the following register-ca-certificate command to register the device root CA as a CA certificate in AWS IoT Core. Make sure to set correct region at --region flag
```

Comment:
Highlight the command, e.g., `register-ca-certificate`, and region flag. Without being set as code, --region becomes and em dash.